README.md

```diff
@@ -302,6 +306,16 @@ program instead of cargo:
 $ cargo watch -x 'test -- --color=always'
 ```
 
+### I want to compile my build with additional features
+
+The `--features` flag is recognized and passed to all cargo invocations.
```

Comment:
Ah, sorry: this is not true anymore :) given the limitations added with your latest changes.